packages/react-sdk/README.md

```diff
@@ -65,6 +65,23 @@ Access the XMTP React SDK [reference documentation](https://xmtp.github.io/xmtp
 
 This client SDK uses a local-first architecture to help you build a production-grade and performant app. To learn more about how we use a local-first architecture, see our [official documentation](https://xmtp.org/docs/build/local-first).
 
+### Database schema updates
+
+When the local DB schema is updated as part of a code change, clients must upgrade their schema version. The default version is `1`.
+
+To specify a new version, use the `dbVersion` prop of the `XMTPProvider` component.
+
+```tsx
+import { XMTPProvider } from "@xmtp/react-sdk";
+import App from "./App.tsx";
+
+export const MyApp: React.FC = () => (
+  <XMTPProvider dbVersion={2}>
```

Comment:
What happens when I increment this version? Is it going to wipe out my entire local DB and force re-create it?

Reply:
just informs Dexie about the new schema so it understands how to work with the data. the current DB and its data should remain intact.

Reply:
@rygine heading over here from [215](https://github.com/xmtp/xmtp-web/issues/215) to further discuss the issue - i've now set the dbVersion to 1 (also tried 2) but on a browser where i did not manually delete the dexie from localStorage i'm getting the following error now: 

![image](https://github.com/xmtp/xmtp-web/assets/80520453/b9555ffa-d5b0-48e9-a2b3-fbbc374c138a)

any ideas?

Reply:
different error when version is set to 2: 
![image](https://github.com/xmtp/xmtp-web/assets/80520453/806ee32d-0635-4545-888c-3f3e53637fe9)

Reply:
when you set the `dbVersion` to `2`, did you have another tab open with the app? i haven't encountered that error yet, but [this issue](https://github.com/dexie/Dexie.js/issues/1706) may provide some insight.

Reply:
no other tab was open in firefox, but the same app dev instance was open in chrome - do you think this messed up the firefox dexie database?
this is the actual error in the console:
![image](https://github.com/xmtp/xmtp-web/assets/80520453/6d78c70e-7a6d-49c3-b9f6-90c4b530bff2)

Reply:
that's strange. chrome shouldn't affect firefox and vice versa.